monitoring/opentelemetry/src/test/java/io/quarkus/ts/opentelemetry/OpenTelemetryDevModeIT.java

```diff
@@ -0,0 +1,93 @@
+package io.quarkus.ts.opentelemetry;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.is;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import io.quarkus.test.bootstrap.DevModeQuarkusService;
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.DevModeQuarkusApplication;
+import io.quarkus.test.services.JaegerContainer;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class OpenTelemetryDevModeIT {
+
+    private static final Logger LOG = Logger.getLogger(OpenTelemetryDevModeIT.class);
+
+    private static final String DISABLE_PROPERTIES = "src/test/resources/disable.properties";
+
+    @JaegerContainer(expectedLog = "\"Health Check state change\",\"status\":\"ready\"")
+    static final JaegerService jaeger = new JaegerService();
+    @DevModeQuarkusApplication
+    static DevModeQuarkusService otel = (DevModeQuarkusService) new DevModeQuarkusService()
+            .withProperty("quarkus.otel.exporter.otlp.traces.endpoint", jaeger::getCollectorUrl);
+
+    @Test
+
+    @Order(1)
+    void checkThereIsNoTreaces() {
+        int pageLimit = 10;
+        String operationName = "GET /hello";
+        await().atMost(5, TimeUnit.SECONDS).untilAsserted(() -> {
+            doRequest();
+            Response response = thenRetrieveTraces(pageLimit, "1h", "pingpong", operationName);
+            response.then()
+                    .body("data", empty());
+        });
+
+    }
+
+    @Test
+    @Order(2)
+    void checkTracesAfterRestart() throws InterruptedException {
+        otel.modifyFile(DISABLE_PROPERTIES,
```

Comment:
Think that `DISABLE_PROPERTIES` file should be add to this PR. Also would be better set this path like this `Paths.get("src", "test", "resources", "disable.properties").toFile().getPath()`. I think it's more safer when working with multiple OS. Or maybe load it as test resource. WDYT?

Reply:
Yes, it's true.